include/revng/BasicAnalyses/AdvancedValueInfo.h

```diff
@@ -463,28 +512,11 @@ class Expression {
     return OperationsStack.at(SmallestRangeIndex).V;
   }
 
-  /// Use LVI to build an expression about \p V
-  ///
-  /// 1. Build a chain of single non-const-operand instructions until you find a
-  ///    phi or a load from a global variable.
-  /// 2. For each instruction in the chain record the number of possible values
-  ///    according to LVI.
-  /// 3. Iterate over the chain looking for the instruction associated with the
-  ///    smallest range.
-  llvm::Instruction *buildExpression(llvm::LazyValueInfo &LVI,
-                                     const llvm::DominatorTree &DT,
-                                     PhiEdges &Edges,
-                                     llvm::Value *V,
-                                     llvm::BasicBlock *StopAt) {
+  /// Builds a chain of single non-const-operand instructions until you find a
+  /// phi or a load from a global variable.
```

Comment:
> or a load from a global variable

Uhm no, until you meet a phi or anything with more than a single non-const-operand.

Reply:
what comment was already there, i just splitted  a larger comment into parts when i splitted the function it referred to.

Reply:
Argh, OK then, I guess.